app/client/reducers/rootReducer.ts

```diff
@@ -42,7 +42,7 @@ const appReducer = combineReducers({
   spreadsheetUpload,
   datasetSave,
   datasetList,
-  newExportRequest,
+  export: ExportReducer,
```

Comment:
wouldn't it make more sense just to follow the common pattern and name the reducer `export`?

Reply:
i guess so, no strong opinion

Reply:
I think the reason I did it was `export` is not allowed and we've done that flow before for previous typescript reducers;

```
  transferUsages: TransferUsageReducer,
  organisations: OrganisationReducer,

```

Reply:
oh export not being allowed is a pretty good justification lol!